x/group/server/msg_server.go

```diff
@@ -425,6 +425,7 @@ func (s serverImpl) CreateProposal(ctx types.Context, req *group.MsgCreatePropos
 		return nil, sdkerrors.Wrap(err, "end time conversion")
 	}
 
+	proposalID := s.proposalTable.Sequence().NextVal(ctx)
```

Comment:
you can just use `s.proposalTable.Sequence().PeekNextVal(ctx)` instead and keep using `s.proposalTable.Create(ctx, m)` which will increment and persist the counter.